### ISSUE TITLE
Undefined control sequence \g_TUDa_dr_bool

### ISSUE THREAD

**Issue by pkonow** [tudace/tuda_latex_templates#121]
Hi, I am trying to get the template running on Fedora 30. I installed the `texlive-tuda-ci` package using `--releasever=31` from the F31 repo. A couple of dependencies were missing in order to compile DEMO-TUDaPub.tex, which I installed manually. The next problematic error I am encountering is:

```
$ lualatex DEMO-TUDaPub.tex
...
(/usr/share/texlive/texmf-dist/tex/latex/base/t1enc.def)))
(/usr/share/texlive/texmf-dist/tex/latex/geometry/geometry.sty
(/usr/share/texlive/texmf-dist/tex/generic/oberdiek/ifpdf.sty)
(/usr/share/texlive/texmf-dist/tex/generic/oberdiek/ifvtex.sty))
(/usr/share/texlive/texmf-dist/tex/latex/koma-script/scrlayer-scrpage.sty
(/usr/share/texlive/texmf-dist/tex/latex/koma-script/scrlayer.sty))
(/usr/share/texlive/texmf-dist/tex/latex/tuda-ci/tudathesis.cfg
! Undefined control sequence.
<argument> \g_TUDa_dr_bool 
                           
l.565 \bool_if:NT \g_TUDa_dr_bool
                                  {
? 
```
Any ideas?

Honestly this entire process of installing missing Texlive dependencies etc. is extremely time-consuming, it would be great to have fully contained distribution of this template that contains all  required Latex packages, all required binaries in the required versions etc. with no dependencies at all (statically compiled for Linux x86_64). :)

**Comment by schoeps** [comment]
> Honestly this entire process of installing missing Texlive dependencies etc. is extremely time-consuming, it would be great to have fully contained distribution of this template that contains all required Latex packages, all required binaries in the required versions etc. with no dependencies at all (statically compiled for Linux x86_64). :)

I guess this was a joke but actually it exists: https://tug.org/texlive/acquire-netinstall.html. In my group most linux users do a full texlive installation like that.

**Comment by TeXhackse** [comment]
@pkonow please drop a complete log file. Otherwhise you can open 1000 issues. Probably it's just something outdated. Same goes for #122 

And as always with LaTeX: Treat one error after another. Because the second one might be indirectly caused by first.

**Comment by pkonow** [comment]
Hi @TeXhackse , sure! :+1:  Here is the log file after encountering the above error:

[DEMO-TUDaPub.log](https://github.com/tudace/tuda_latex_templates/files/3962211/DEMO-TUDaPub.log)

**Comment by TeXhackse** [comment]
Yup. As expected. 

Tex Live 2018 works if you have a clean installation  (+ Tex Live 2018 also contains some known bugs.). Some linux distributions mix up different versions of packages.

Anyway. Easiest way is to get a TexLive 2019 or newer. And I know you are able to do that, especially as a Fedora user.

We discussed to be backwards compatible for some years, but as we cannot create custom packages for every Linux distribution we stopped that at the basics. Especially since they often mix up different versions.

**Comment by pkonow** [comment]
Ok, I installed the full standalone TexLive 2019 suite and now it's working perfectly. This was totally my bad since this is explicitly mentioned on the homepage at https://www.ce.tu-darmstadt.de/ce/latex_tuda/index.en.jsp:

> In linux you should install a current TeXlive from the bundles available at www.tug.org. This ensures installation of most recent package versions. (In some cases problems were reported when using tuda-ci within a TeX[live] system, that has been installed using the package management of the respective linux distribution.)

I guess I got confused by the existence of the tuda-ci package in the Fedora repository. Maybe this warning should be highlighted more boldly since as a Linux user it is generally preferred to choose the packaged version of the Linux distribution whenever possible. The `install-tl` script from tug.org seems to be pretty solid and is clearly the preferred installation method for Linux users IMO.

Thanks for your work and for the immediate responses!
